ArcGISRuntimeSDKQt_CppSamples/Maps/CreateDynamicBasemapGallery/BasemapStyleListModel.h

```diff
@@ -0,0 +1,39 @@
+#ifndef BASEMAPSTYLELISTMODEL_H
+#define BASEMAPSTYLELISTMODEL_H
+
+#include <QAbstractListModel>
+#include <QObject>
+#include <QUrl>
+
+struct StylePreview {
```

Comment:
`BasemapStyleInfo` objets used instead of custom `StylePreview` struct in https://github.com/Esri/arcgis-maps-sdk-samples-qt/pull/1714/commits/f017d0c994b7d7ba611ed7185fdae9e884ce1dba.